itest.py

```diff
@@ -0,0 +1,832 @@
+#! /usr/bin/env python3
```

Comment:
please add words about expected inputs and outputs for this script.

Reply:
Is it okay to put this in a `--help`, or should it (also) be here at the top?

Reply:
`--help` is fine.     easy to copy/paste to the most helpful place later, as long as it exists.